### PR TITLE
Update _config.py

### PR DESCRIPTION
**Pull request by oencarnacion**
New reports added to the config file from Shadowserver: 

open_mysql
open_smtp
open6_ssh
open6_smtp
scan_quic
open6_ssl
 open_amqp
scan_ddos_middlebox
open6_telnet
scan_ics
scan_epmd
open_scan_dvr_dhcpdiscover
scan_kubernetes
scan_socks
scan_ssh
device_id
accessible_ssl

# Please read the notes below and replace them with the description of you pull request

Make sure you follow the instructions in the [Developer Guide](https://intelmq.readthedocs.io/en/latest/dev/guide.html) - it describes how to run the test suite and which coding rules to follow.

# Commits

Please review your commits before creating a pull request. We try to keep our commit history clean.
If you had to fix something in your code and added another commit on top of your existing work, please [squash](https://blog.simontimms.com/2016/02/18/i-squash-my-pull-requests-and-you-should-too/) your commits.
This makes it a lot easier to find out why things work the way they do later on.

But please also keep in mind that one commit should only contain changes that belong together.
> The cardinal rule for creating good commits is to ensure there is only one "logical change" per commit
> - [OpenStack Wiki on GitCommitMessages](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

Thus, if your changes affect multiple files and also contains tests and documentation (which it should), it might make sense to have the tests and the documentation in a separate commit.

# Commit Messages

Please explain your changes also in the git commit message.
The commit message should contain a subject that gives an overview of the changes in the commit and is limited to 72 characters.
It should start with a capital letter and it should not end with a period.
Below the subject should be, separated by an empty line, the body of the commit message. The body should explain what the commit changes and why it changes thing the way it does.
Explain your modification and also explain why you didn't chose a different approach.
See also [How To Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).

# Description

If you are following the commit message guidelines above, all the relevant information should already be part of the commit message.
If there is anything else you want to add, feel free to do this here.

### CODE REVIEW

**Comment by oencarnacion** [comment]
new reports added to the config file from shadowserver

File: intelmq/bots/parsers/shadowserver/_config.py [hunk 1]
```diff
@@ -3040,8 +3543,7 @@ def scan_exchange_identifier(field):
     ('Outdated-DNSSEC-Key', 'outdated_dnssec_key', outdated_dnssec_key),
     ('Outdated-DNSSEC-Key-IPv6', 'outdated_dnssec_key_v6', outdated_dnssec_key),
     ('SSL-FREAK-Vulnerable-Servers', 'scan_ssl_freak', ssl_freak_vulnerable_servers),
-    ('SSL-POODLE-Vulnerable-Servers IPv4', 'scan_ssl_poodle', ssl_poodle46_vulnerable_servers),
-    ('SSL-POODLE-Vulnerable-Servers IPv6', 'scan6_ssl_poodle', ssl_poodle46_vulnerable_servers),
+    ('SSL-POODLE-Vulnerable-Servers', 'scan_ssl_poodle', ssl_poodle_vulnerable_servers),
```
Comment (sebix) [inline intelmq/bots/parsers/shadowserver/_config.py:3546]: Why did you remove the IPv4/IPv4 file name mappings here? Did shadowserver stop sending reports per IP protocol and just one for both v4 and v6?

File: intelmq/bots/parsers/shadowserver/_config.py [hunk 2]
```diff
@@ -2998,8 +3486,6 @@ def scan_exchange_identifier(field):
     ('Compromised-Website', 'compromised_website', compromised_website),
     ('DNS-Open-Resolvers', 'scan_dns', dns_open_resolvers),
     ('Darknet', 'darknet', darknet),  # legacy (replaced by event4_honeypot_darknet)
-    ('Device-Identification IPv4', 'device_id', device_id),
-    ('Device-Identification IPv6', 'device_id6', device_id),
```
Comment (sebix) [inline intelmq/bots/parsers/shadowserver/_config.py:3002]: Same comment as with Poodle: Is this a change by Shadowserver?

**Comment by oencarnacion** [comment]
Hi Sebastian,

I just confirmed and Shadowserver still sends those reports. It was my mistake and I just made the corrections. I'll attach the file to you this way and request your approval again in the repository.



Analista Programador CSIRT-RD
Centro Nacional de Ciberseguridad
Ministerio de la Presidencia | Presidencia de la República
Cel: 849-655-1482
Flot: 829-222-0131
Correo: ***@***.******@***.***>
URL: www.cncs.gob.do<http://www.cncs.gob.do/>

***@***.******@***.******@***.***
***@***.***
La información contenida en este mensaje y/o en los archivos adjuntos es confidencial y está dirigida al uso exclusivo del emisor y/o de la persona o entidad a quien va dirigida. Si usted no es el destinatario, cualquier almacenamiento, divulgación, distribución o copia de esta información está estrictamente prohibida y sancionada por la ley. Si usted recibe este mensaje y/o archivos adjuntos por error, por favor notifique inmediatamente al remitente y borre de su sistema el mensaje y todos los archivos adjuntos. Gracias.

The information in this e-mail and/or attachments is intended to be confidential and only for use of the individual or entity to whom it is addressed and/or the issuer. If you are not the intended recipient, any retention, dissemination, distribution or copying of this message is strictly prohibited and sanctioned by law. If you receive this message and/or attachments in error, please notify the sender immediately and delete this message and all its attachments from your computer. Thank you.

**Comment by sebix** [comment]
@elsif2 for review

**Comment by elsif2** [comment]
The above changes are superseded by https://github.com/certtools/intelmq/pull/2143.

**Comment by oencarnacion** [comment]
Hi @***@***.***>

Some status on the shadowserver parser bot contribution I made last week?


Analista Programador CSIRT-RD
Centro Nacional de Ciberseguridad
Ministerio de la Presidencia | Presidencia de la República
Cel: 849-655-1482
Flot: 829-222-0131
Correo: ***@***.******@***.***>
URL: www.cncs.gob.do<http://www.cncs.gob.do/>

***@***.******@***.******@***.***
***@***.***
La información contenida en este mensaje y/o en los archivos adjuntos es confidencial y está dirigida al uso exclusivo del emisor y/o de la persona o entidad a quien va dirigida. Si usted no es el destinatario, cualquier almacenamiento, divulgación, distribución o copia de esta información está estrictamente prohibida y sancionada por la ley. Si usted recibe este mensaje y/o archivos adjuntos por error, por favor notifique inmediatamente al remitente y borre de su sistema el mensaje y todos los archivos adjuntos. Gracias.

The information in this e-mail and/or attachments is intended to be confidential and only for use of the individual or entity to whom it is addressed and/or the issuer. If you are not the intended recipient, any retention, dissemination, distribution or copying of this message is strictly prohibited and sanctioned by law. If you receive this message and/or attachments in error, please notify the sender immediately and delete this message and all its attachments from your computer. Thank you.

**Comment by sebix** [comment]
@elsif2 Reminder on this PR as well.

**Comment by elsif2** [comment]
Thank you for your contribution.  This PR has been obsoleted by recent updates. All Shadowserver reports are now supported in the current develop branch and will be included in the 3.1.0 release.

**Comment by oencarnacion** [comment]
and what about the changes applied shadowserver 2022-06 that I just contributed?

Enviado desde mi Samsung Mobile de Claro
Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by elsif2** [comment]
The report types added in your 2022-06 contribution have already been added to the develop branch though a separate PR.
